src/javascript/weather.js

```diff
@@ -0,0 +1,200 @@
+/* eslint-disable consistent-return */
+import UnsplashBG from './unsplash';
+import weatherInfo from './weatherInfo';
+import loaderAnimation from './loaderAnimation';
+
+const weatherKey = process.env.WEATHER_API;
+
+const imgBG = new UnsplashBG();
+
+function citiesListCont() {
```

Comment:
### Uses `ES6` syntax 

* [ ] Kindly ensure that you are using ES6 syntax consistently, all across the codebase as explained in this ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)). Here you are supposed to use arrow function instead of the `function` keyword in this  `citiesListCont` declaration.

```suggestion
const citiesListCont = () => {
```